gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -20,6 +20,7 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
 
   OrchestrationTemplateCfn.register_eligible_manager(self)
   ExtManagementSystem.register_cloud_discovery_type('amazon' => 'ec2')
+  MiqProvision.register_supported_ems(self)
```

Comment:
@Fryguy I'm not sure if this is the right place to do all the `register_` stuff. When is a `CloudManager` constant loaded the first time? Not at bootstrap time! But will it be loaded before `app/models/miq_provision/options_helper.rb` is used?

Maybe we want to reconsider eager_loading pluggable providers :smile: ???